dev_scripts/translation.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+const fs = require('fs');
+
+const localizationFiles = [];
+const nextLine = /{nextLine}/g;
+const selectedLanguage = process.argv[3];
+const newFileFolder = './src/localization/translations/';
+const message =
+  '\nMore information about this script here: https://github.com/openmsupply/mobile/wiki/Scripts\n';
+
+// Reads all JSON files in localization files and save them in localizationFiles array
+const filesFolder = './src/localization/';
+fs.readdirSync(`./${filesFolder}`).forEach(file => {
+  if (file.split('.').pop() === 'json') {
+    localizationFiles.push({
+      fileContent: require(`../${filesFolder}${file}`),
+      fileName: file.replace('.json', ''),
+    });
+  }
+});
+
+// Exports all strings from the selectedLanguage to CSV
+// csvData keeps all the content in csv formal
+const exportTranslation = () => {
+  let csvData = '';
+  localizationFiles.forEach(localizationFile => {
```

Comment:
could do (also not default language in constant pls):
```
localizationFiles.forEach({fileContent, fileName} => {
...
const selectedLanguageContent = fileContent[selectedLanguage];
const mainLanguageContent = fileContent[mainLanguage]
Object.entries(mainLanguageContent) ...
```
FYI, interesting syntax: `const { [selectedLanguage]: selectedLanguageContent, [mainLanguage]: mainLanguageContent } = fileContent`

Reply:
Applied and working well! 
```
 localizationFiles.forEach(({ fileName, fileContent }) => {
    const {
      [mainLanguage]: mainLanguageContent,
      [selectedLanguage]: selectedLanguageContent,
    } = fileContent;
```